README.md

```diff
@@ -211,7 +211,27 @@ In the example below,
 
 ![annotation output](https://raw.githubusercontent.com/ovh/ovh-warp10-datasource/master/dist/assets/screenshot-customAllValue.png)
 
+## Query returning Labels
 
+A variable can contain values for a defined Label. For example, to get all the unique values for the key `hostname`, you can specify a query like this in the templating variable *Query* setting.
+```
+[ $ReadToken '~.*' { } ] FIND
+<% DROP LABELS 'hostname' GET %> LMAP
+UNIQUE
```

Comment:
Most of the time, you can add `LSORT` at the end

Reply:
Sort can also be made within Grafana variable declaration itself.
Let's keep the script as simple as possible.